homeassistant/components/homekit_controller/__init__.py

```diff
@@ -255,19 +255,23 @@ def update(self):
         # pylint: disable=import-error
         from homekit.exceptions import AccessoryDisconnectedError
 
+        pairing = self._accessory.pairing
+
         try:
-            pairing = self._accessory.pairing
-            data = pairing.list_accessories_and_characteristics()
+            new_values_dict = pairing.get_characteristics(self._chars_to_poll)
         except AccessoryDisconnectedError:
             return
-        for accessory in data:
-            if accessory['aid'] != self._aid:
+
+        for (_, iid), result in new_values_dict.items():
+            if 'value' not in result:
                 continue
-            for service in accessory['services']:
-                if service['iid'] != self._iid:
-                    continue
-                self.update_characteristics(service['characteristics'])
-                break
+            # Callback to update the entity with this characteristic value
+            char_name = escape_characteristic_name(self._char_names[iid])
+            update_fn = getattr(self, '_update_{}'.format(char_name), None)
+            if not update_fn:
+                continue
+            # pylint: disable=E1102
```

Comment:
Please write out the pylint disable code in text. I think this is `not-callable`.